Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM node:10.9-slim as builder
+
+RUN apt-get update && apt-get install -y \
+    build-essential \
+    g++ \
+    gcc \
+    git \
+    python2.7 \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN ln -s /usr/bin/python2.7 /usr/bin/python2
+
```

Comment:
Does tutor-js need python?

Reply:
some of the npm modules build themselves and need it.. i forget which one